lib/MusicBrainz/Server/Form/Role/ValidateUsername.pm

```diff
@@ -0,0 +1,22 @@
+package MusicBrainz::Server::Form::Role::ValidateUsername;
+use HTML::FormHandler::Moose::Role;
+use MusicBrainz::Server::Translation qw( l );
+
+sub validate_username
+{
+    my ($self, $field) = @_;
+
+    my $username = $field->value;
+    if (defined $username) {
+        if ($username =~ qr{^deleted editor \#\d+$}i) {
+            $field->add_error(l('This username is reserved for internal use'));
```

Comment:
Well, if it's "sometimes", let's move towards more consistently using proper grammar than now. :)